pkg/api/unversioned/helpers.go

```diff
@@ -181,3 +184,34 @@ func ExtractGroupVersions(l *APIGroupList) []string {
 	}
 	return groupVersions
 }
+
+// ExtractGroupVersionResourcesWithVerbs extracts the GroupVersionResources from a map. In case of
+// errors, an aggregate error is returned and those resources which could be parsed successfully.
+func ExtractGroupVersionResources(rls []*APIResourceList) ([]schema.GroupVersionResource, error) {
```

Comment:
> Die or getting moved and renamed.

Death can be a followup.  I don't care about moving it since I want to kill it.  I also don't want to extend the pattern I don't like for functions I'm likely to keep.